install/resources/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# this file was generate by jluna/install/init.sh
```

Comment:
This entire file has issues pretty much on every line. Revise based on previous comments.